client/src/main/java/com/arcbees/beeshop/client/events/BrandChangedEvent.java

```diff
@@ -0,0 +1,31 @@
+package com.arcbees.beeshop.client.events;
```

Comment:
Ok, I'll create a task to add it to every file that needs it. Will do the actual work in another PR.